### ISSUE TITLE
Difference between TweenedScreen and SimpleTweenScreen

### ISSUE THREAD

**Issue by wisdom-githb** [Xerios/ScreenManager#17]
Hi, 

I just wanted to know the difference between these two screens, and how does it really affect if I use one instead of other. 

Edit- My use case is that I wanted to create different selection panels before starting the game. All of the selection panels are full canvas size panels. With a flow something like this-

![image](https://user-images.githubusercontent.com/25106770/164568817-65eac750-d21c-4e29-b717-4a32215d67ed.png)

I wanted to be able to load and go back from screens accordingly

(I did not know how to add the  "how-to" label since this is not an issue, rather just a question)

**Comment by wisdom-githb** [comment]
Its just a difference of animation of how the UI elements are loaded. I just ended up trying things and figuring it out. 
I ran into another issue which I will open newly for better maintainability.
